Videos/videos_info.xml

```diff
@@ -242,18 +242,53 @@
         <date>2023-06-14</date>
         <renesaslink>https://www.renesas.com/video/how-use-smart-configurator-walkthrough-settings-clocks-and-components</renesaslink>
     </info>
-    <info id="_Toc87439381" name="videoInfo" device="RX || RL78 || RH850 || COMMON" category="tutorials">

```

Comment:
Q: Why is this removed?

Reply:
This video is removed from RH850 and Common pages as it is no longer listed in the excel sheet.